releasenotes/README.md

```diff
@@ -33,9 +33,17 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
 ------------
 
 * NEW: Plugin installs use `yarn` under the hood. If you run into errors during installs or updates, you can now enable the network mutex option by setting the SF_USE_NETWORK_MUTEX environment variable to `true`. Setting this variable opens a local network to manage the concurrent `yarn` instances and may be more reliable. You can also pass an optional port for the local server to open on with the SF_NETWORK_MUTEX_PORT environment variable.  See the [yarn documentation](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-concurrency-and-mutex) for more information. [oclif plugin-plugins PR [#670](https://github.com/oclif/plugin-plugins/pull/670))
+
+* NEW: Poll for status of a deployment when you run `project deploy report` with the new `--wait` flag. If you specify this flag, the command polls for the status every second until the timeout of `--wait` minutes.  If you don't specify the `--wait` flag, the command simply checks and displays the status of the deploy; the command doesn't poll for the status.
+
+    We also added the `--target-org` flag to `project deploy report`.  You usually don't need to specify this flag because the cached deploy job already references the org to which you deployed.  But if you run `project deploy report` on a computer different than the one from which you originally deployed, then you must specify the `--target-org` and it must point to the same org. If it doesn't, you get an error. 
```

Comment:
Small nit-pick: If a default org is specified in the project for the other computer and it matches the org that was deployed to, then they won't have to set `--target-org`